source/localizable/routing/specifying-a-routes-model.md

```diff
@@ -23,7 +23,7 @@ export default Ember.Route.extend({
 ```
 
 Typically, the `model` hook should return an [Ember Data](../../models/) record,
-but it can also return any [promise](https://www.promisejs.org/) object (Ember
```

Comment:
I had to capitalize because it said *promise object*. I thought of it along the same lines as, say, an `EmberObject` object. If it said "but it can also return a promise" then I think it could stay that way.